modules/RTC/RTCUtils.js

```diff
@@ -119,6 +119,12 @@ function getConstraints(um, options) {
             constraints.video.optional.push({
                 sourceId: options.cameraDeviceId
             });
+        } else {
+            // by default prefer the "user" camera if we have not specified the
+            // exact device id for mobile devices
+            if (RTCBrowserType.isReactNative() || RTCBrowserType.isAndroid()) {
+                constraints.video.facingMode = "user";
```

Comment:
Well, it's not yet supported in Chrome, but it doesn't break anything. So probably you are right, let's  make it for all browsers.